Configuration/Geometry/README.md

```diff
@@ -94,11 +99,13 @@ Several detector combinations have been generated:
 * D68 = T21+C11+M6+I11+O5+F4 (For HGCAL study on evolution of detector)
 * D70 = T21+C13+M7+I11+O6+F6 (For HGCAL study on evolution of detector)
 * D76 = T21+C14+M9+I13+O7+F6
-* D77 = T24+C14+M9+I13+O7+F6
+* D77 = T24+C14+M9+I13+O7+F6 (Current default scenario)
 * D78 = T22+C14+M9+I13+O7+F6
 * D79 = T23+C14+M9+I13+O7+F6
 * D80 = T25+C14+M9+I13+O7+F6
 * D81 = T26+C14+M9+I13+O7+F6
 * D82 = T21+C15+M9+I13+O7+F7
 * D83 = T24+C16+M9+I13+O7+F6
 * D84 = T24+C13+M7+I11+O6+F6 (For HGCAL study on evolution of HGCal replacing D70)
+* D85 = T24+C14+M9+I14+O7+F6
+* D86 = T24+C17+M10+I14+O8+F6
```

Comment:
This will conflict with the geometry introduced at https://github.com/cms-sw/cmssw/pull/34120.
@adewit FYI

Reply:
I understand this only now. I did not realize that T27 is put in till I submitted this PR. If #34120 is merged, I shall take proper action

Reply:
For D86, do you want to jump both Calorimeter and Muon, not one-by-one as we usually do?